packages/angular/src/migrations/update-15-9-0/update-testing-tsconfig.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  createProjectGraphAsync,
+  getProjects,
+  joinPathFragments,
+  Tree,
+  updateJson,
+} from '@nrwl/devkit';
+import { forEachExecutorOptionsInGraph } from '@nrwl/devkit/src/generators/executor-options-utils';
+import type { JestExecutorOptions } from '@nrwl/jest/src/executors/jest/schema';
+
+export async function updateTestingTsconfigForJest(tree: Tree) {
+  const graph = await createProjectGraphAsync();
+  const projects = getProjects(tree);
+  forEachExecutorOptionsInGraph<JestExecutorOptions>(
+    graph,
+    '@nrwl/jest:jest',
+    (options, projectName) => {
+      const projectConfig = projects.get(projectName);
+
+      if (!isJestPresetAngular(tree, options.jestConfig)) {
+        return;
+      }
+      // TODO(caleb): can we assume this?
+      const tsconfigPath = joinPathFragments(
+        projectConfig.root,
+        'tsconfig.spec.json'
+      );
+
+      if (tree.exists(tsconfigPath)) {
+        updateJson<TsConfig>(
+          tree,
+          tsconfigPath,
+          (json) => {
+            json.compilerOptions ??= {};
+            json.compilerOptions.target ??= 'es2016';
```

Comment:
Is there a reason we're using es2016 and not es2022 which Angular apps were migrated to in Version 15?

Reply:
> Angular doesn't support native async/await in testing with target higher than ES2016, see https://github.com/angular/components/issues/21632#issuecomment-764975917

https://github.com/thymikee/jest-preset-angular#migration-from-angular--13